scripts/external_data/load_statcast.py

```
@@ -29,63 +29,392 @@ def get_conn():
+            api_break_x_batter_in TEXT,
+            arm_angle TEXT,
+            attack_angle TEXT,
+            attack_direction TEXT,
+            swing_path_tilt TEXT,
+            intercept_ball_minus_batter_pos_x_inches TEXT,
+            intercept_ball_minus_batter_pos_y_inches TEXT
         )
     """)
 
 def copy_to_staging(cur, csv_path):
     with open(csv_path, "r", newline="") as f:
```

Ruff (0.15.10)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)